rl/patches/sitecustomize.py

```
@@ -0,0 +1,18 @@
+is added to PYTHONPATH by env.rjob.sh, so the patch is applied before
+slime/train.py runs — no code change to slime or Megatron needed.
+"""
+# Import any pre-existing sitecustomize first (chained sitecustomize).
+try:
+    import _orig_sitecustomize  # noqa: F401
+except Exception:
+    pass
+
+try:
+    import gdn_packed_seq  # noqa: F401  — applies the monkey-patch
+except Exception as _e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)